cli/cmd/docs/open.go

```diff
@@ -0,0 +1,24 @@
+package docs
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+var docsURL = "https://docs.rilldata.com"
+
+func OpenCmd() *cobra.Command {
+	docsCmd := &cobra.Command{
+		Use:   "open",
+		Short: "Open docs.rilldata.com",
+		Run: func(cmd *cobra.Command, args []string) {
+			err := browser.Open(docsURL)
+			if err != nil {
+				fmt.Printf("Could not open browser. Copy this URL into your browser: %s\n", docsURL)
+			}
+		},
+	}
+	return docsCmd
+}
```

Comment:
This should still happen just on `rill docs` if possible. If not possible, we should probably move the generator to `scripts/generate_cli_docs` or something.